pkg/endpointmanager/policymap_pressure.go

```diff
@@ -73,7 +73,7 @@ func (mgr *policyMapPressure) update() {
 
 	mgr.RLock()
 	max := float64(0)
-	for _, value := range maps.Values(mgr.current) {
+	for value := range maps.Values(mgr.current) {
```

Comment:
Had to double-check that this was correct, since the types work out the same. But yes, this is good :-).